ledger-core/v2/virtual/virtual.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package virtual
+
+import (
+	"context"
+	"time"
+
+	"github.com/insolar/component-manager"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/configuration"
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor"
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/flow/dispatcher"
+	lrCommon "github.com/insolar/assured-ledger/ledger-core/v2/logicrunner/common"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/statemachine"
+)
+
+type Virtual interface {
```

Comment:
what exactly does this thing is virtual.Virtual? :octocat:

Reply:
This is Draft PR, so interface is empty (for now)

Reply:
As discussed, this should be virtual.AppDispatcher